Zotero/Scenes/Detail/Items/Views/ItemsViewController.swift

```diff
@@ -684,66 +686,64 @@ final class ItemsViewController: UIViewController {
     @discardableResult
     private func setupSearchBar(for windowSize: CGSize) -> SearchBarPosition {
```

Comment:
It's outside of scope of this, but I think we can finally move away from this terrible search bar placement and move it to the iOS default which is in navigation bar now. If you don't want to do it here feel free to create a ticket for it.

Reply:
Opened https://github.com/zotero/zotero-ios/issues/764